packages/x-charts/src/ChartsOnAxisClickHandler/ChartsOnAxisClickHandler.tsx

```diff
@@ -6,7 +6,7 @@ import { useSeries } from '../hooks/useSeries';
 import { useSvgRef } from '../hooks';
 import { useCartesianContext } from '../context/CartesianProvider';
 
-type AxisData = {
+export type AxisData = {
```

Comment:
If we export this, we should likely prefix it to guarantee unicity in the namespace.

```suggestion
export type ChartsAxisData = {
```